ieeeuoft/makeuoft-site-2021
# Update Application Fields

## Overview

- Resolves #14
- Updates the application model/form fields to be those in #14
    - Including code of conduct link, should appear on the form
    - The school field shows as "University" now
- Adds a birthday validator that they must be over 18
- Minor updates to forms on the admin site (applications and team review)

## Unit Tests Created
- Test that birthday validator works
- Updates the data passed to the application form everywhere it's needed

## Steps to QA
- Update the date in the settings file so you can submit an application. Create an account if necessary
- Try all the failure cases for the application form:
    - Missing fields (everything but resume sharing required)
    - Birthday after feb 6 2003 (18)
- Make sure the code of conduct link in the form works
- Make sure that the admin site forms for application and team review are still working correctly
